airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -651,6 +651,19 @@ def get_partitions(
                 pnames = [p.name for p in table.partitionKeys]
                 return [dict(zip(pnames, p.values)) for p in parts]
 
+    def get_partition_by_name(self, db_name: str, tbl_name: str, part_name: str) -> Any:
+        """Get a metastore partition object
+
+        >>> hh = HiveMetastoreHook()
+        >>> tp = hh.get_table(db_name='org_db', tbl_name='my_year_partitioned_table', part_name='year=2020')
+        >>> tp.parameters.get("totalSize")
+        '713263'
+        >>> tp.sd.location
+        'hdfs://foo/bar'
```

Comment:
The example shows: `HiveMetastoreHook().get_table`, is that intentional?

I assume it should be `HiveMetastoreHook(). get_partition_by_name`

Reply:
Can you also add a simple test to make sure that `client.get_partition_by_name` is call with correct args